docs/source/python/extending_types.rst

```diff
@@ -131,58 +131,82 @@ and serialization mechanism. The extension name and serialized metadata
 can potentially be recognized by other (non-Python) Arrow implementations
 such as PySpark.
 
-For example, we could define a custom UUID type for 128-bit numbers which can
-be represented as ``FixedSizeBinary`` type with 16 bytes::
+For example, we could define a custom rational type for fractions which can
+be represented as a pair of integers::
 
-    class UuidType(pa.ExtensionType):
+    import pyarrow as pa
+    import pyarrow.types as pt
+
```

Comment:
I think it's better to:
- omit these imports from the example
- just use `pa.types` in the code (instead of `pt`) here and in the other files

Reply:
Done